config/runtime/error_handler.go

```diff
@@ -21,48 +25,65 @@ func newErrorHandler(ctx *hcl.EvalContext, opts *protectedOptions, log *logrus.E
 		if !ok {
 			continue
 		}
+
+		handlersPerKind := make(map[string]*config.ErrorHandler)
 		for _, h := range definition.ErrorHandler {
 			for _, k := range h.Kinds {
-				if _, exist := kindsHandler[k]; exist {
-					return nil, fmt.Errorf("error handler type already exists: " + k)
-				}
-
-				contextBody := h.HCLBody()
+				handlersPerKind[k] = h
+			}
+		}
 
-				epConf := &config.Endpoint{
-					Remain:    contextBody,
-					Proxies:   h.Proxies,
-					ErrorFile: h.ErrorFile,
-					Requests:  h.Requests,
-					Response:  h.Response,
+		// this works if wildcard is the only "super kind"
+		if mappedKinds, mkExists := wildcardMap[ref]; mkExists {
+			// expand wildcard:
+			// * set wildcard error handler for mapped kinds, no error handler for this kind is already set
+			// * remove wildcard error handler for wildcard
+			if wcHandler, wcExists := handlersPerKind["*"]; wcExists {
```

Comment:
I see here the handling of `error_handler "*" {...}`, but not for `error_handler {...}`. But it works. Why? :)

Reply:
config/configload/error_handler.go:
```go
func newKindsFromLabels(block *hcl.Block) ([]string, error) {
...
	if len(allKinds) == 0 {
		allKinds = append(allKinds, errors.Wildcard)
	}
```